src/selectors/layers.js

```diff
@@ -10,41 +10,35 @@ const getLayerName = state => state.layer.name;
 export const getLayers = createSelector(
   [getLayoutNodes, getVisibleLayerIDs, getLayerName, getGraphSize],
   (nodes, layerIDs, layerName, { width }) => {
-    // Get list of layer Y positions from nodes
-    const layerY = nodes.reduce((layerY, node) => {
-      if (!layerY[node.layer]) {
-        layerY[node.layer] = [];
-      }
-      layerY[node.layer].push(node.y);
-      return layerY;
-    }, {});
+    const bounds = {};
```

Comment:
While implementing layers in the new layout algorithm there were some cases the original function appeared to not correctly handle causing overlaps. This replacement function didn't seem to have those problems I was having at the time,  but it may well have just been that those were invalid cases anyway, it's hard to tell.

Either way I think this version might be a little more performant than the original even if equivalent, since it does min/max once per layer vs twice and uses less memory (`O(layers * 2)` vs `O(nodes)`), if I understand both correctly?

Might be worth keeping if you agree, otherwise I can easily revert it unless I see the original problem again.